### DISCUSSION TITLE
reconnect to another ap

### DISCUSSION THREAD

**Discussion by Abedi98** [Hieromon/AutoConnect#412] [Q&A]
When the default access point is no longer available, how can I connect to previously active access points without resetting?

**Comment by Hieromon** [comment]
Try [`AutoConnectConfig::autoReconnect`](https://hieromon.github.io/AutoConnect/adconnection.html#automatic-reconnect) setting. It will select one of the receivable SSIDs that has established a connection in the past and attempts to reconnect.

#### PLEASE NOTE
From now on, please post this kind of question to [**Discussions**](https://github.com/Hieromon/AutoConnect/discussions). It is a bulletin board suitable for this kind of posting, and you can expect advice from experts other than me.
And, your question may provide useful help to other users. The issue topic will eventually close, so other users may miss it.

Thank you.
